recipes/p4api/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "p4api" %}
+{% set long_version = "2019.1.1796703" %}
+{% set version = "19.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ long_version }}"
+
+source:
+  url: http://ftp.perforce.com/perforce/r{{ version }}/bin.linux26x86_64/p4api.tgz   # [linux]
+  sha256: 83a365fe190ddbdb41d40ba230e765bcdd9fb7bf57934812d74a4b4cfbf72349           # [linux]
+  url: http://ftp.perforce.com/perforce/r{{ version }}/bin.darwin90x86_64/p4api.tgz  # [osx]
+  sha256: 68fb0b57bc7db95ca0c08a5b84e57dc1d82b6dec0b2723fc48a181887f280485           # [osx]
+  url: http://ftp.perforce.com/perforce/r{{ version }}/bin.mingw64x86/p4api.zip      # [win]
+  sha256: 342bfc820a988db11002ff1804f404796f06b214ba7630d6d8a3593aa564b7bf           # [win]
+
+build:
+  number: 0
+  script:
+    - cp -r include ${PREFIX}  # [unix]
+    - cp -r lib ${PREFIX}      # [unix]
+    - cp -r include %PREFIX%   # [win]
+    - cp -r lib %PREFIX%       # [win]
+
+requirements:
+  run:
+    - openssl
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libclient.a       # [unix]
+    - test -f ${PREFIX}/lib/libp4script.a     # [unix]
+    - test -f ${PREFIX}/lib/libp4script_c.a   # [unix]
+    - test -f ${PREFIX}/lib/librpc.a          # [unix]
+    - if not exist %PREFIX%\\Library\\LIB\\libclient.a       # [win]
+    - if not exist %PREFIX%\\Library\\LIB\\libp4script.a     # [win]
+    - if not exist %PREFIX%\\Library\\LIB\\libp4script_c.a   # [win]
+    - if not exist %PREFIX%\\Library\\LIB\\librpc.a          # [win]
+
+about:
+  home: http://www.perforce.com
+  license: Freely Distributable 
```

Comment:
I am not sure what is the right license I should write here

Reply:
Looks like a BSD-2-Clause license, no?

Reply:
Indeed! Looks like it, I'm going to change it
Thanks! :)